lua/orgmode/utils/init.lua

```diff
@@ -6,7 +6,8 @@ local debounce_timers = {}
 local query_cache = {}
 local tmp_window_augroup = vim.api.nvim_create_augroup('OrgTmpWindow', { clear = true })
 
-function utils.readfile(file)
+function utils.readfile(file, opts)
+  opts = vim.tbl_deep_extend('keep', opts or {}, { raw = false })
```

Comment:
Doing this should be enough, since `nil` is considered a falsy value.
```suggestion
  opts = opts or {}
```